packages/@aws-cdk/cdk/lib/core/construct.ts

```diff
@@ -391,6 +446,18 @@ export class ConstructNode {
  * another construct.
  */
 export class Construct implements IConstruct {
+  /**
+   * Run the prepare phase on the given construct
+   */
+  public static doPrepare(construct: IConstruct) {
```

Comment:
I want `prepare()` to be protected because I don't want it to show up when you hit `.` when typing

```ts
bucket.
```

It's not a user-facing capability.

This method gives us type checking, which "as any" does not.